app/controllers/api/auth/accounts_controller.rb

```diff
@@ -7,6 +7,15 @@ class Api::Auth::AccountsController < Api::AccountsController
 
   represent_with Api::Auth::AccountRepresenter
 
+  filter_resources_by :user_id
+
+  def create_resource
+    return super unless params['user_id']
```

Comment:
We want to override the default resource creation logic if the account is being created through the `/api/v1/auth/users/:user_id/account` endpoint.